openassessment/templates/openassessmentblock/grade/oa_grade_complete.html

```diff
@@ -144,7 +144,7 @@ <h4 id="feedback__overall__prompt__{{ xblock_id }}" class="field-group__label">{
                                                            class="option__input feedback__overall__value"
                                                            value="These assessments were useful."
                                                            aria-labelledby="feedback__overall__prompt__{{ xblock_id }}"/>
-                                                    <label class="option__label" for="feedback__overall__value--useful">{% trans "These assessments were useful." %}</label>
+                                                    <label class="option__label" for="feedback__overall__value--useful__{{ xblock_id }}">{% trans "These assessments were useful." %}</label>
```

Comment:
@staubina FYI, this is how i18n works in Django Templates in general:

https://docs.djangoproject.com/en/1.10/topics/i18n/translation/#internationalization-in-template-code